kubernetes/deployment/out-of-tree/deployment.yaml

```
@@ -13,7 +13,7 @@ spec:
     spec:
       containers:
       - name: machine-controller-manager
-        image:eu.gcr.io/gardener-project/gardener/machine-controller-manager:v0.28.0-dev-793b105c41adfc434e07107decaf1ac67fc1fd3f
+        image: eu.gcr.io/gardener-project/gardener/machine-controller-manager:v0.28.0-dev-793b105c41adfc434e07107decaf1ac67fc1fd3f
```

```suggestion
        image: eu.gcr.io/gardener-project/gardener/machine-controller-manager:v0.39.0
```
Aligning the image with the latest stable MCM image.